### DISCUSSION TITLE
Beats applications questions, and clients not reporting to SO server.

### DISCUSSION THREAD

**Discussion by ll3N1GmAll** [Security-Onion-Solutions/securityonion#10343] [Unsupported Versions]
I have installed the "Beats" files (audit, file, and metric) on multiple linux systems and configured their respective .yml config files to enable logstash and point the clients to our standalone SO server, but I'm not seeing anything reporting in. I have installed Winlogbeat on multiple windows systems, and they are all reporting in successfully.
So-allow has been run to allow the traffic to come in to the server and the firewalls on the clients are open to and from the clients to the SO server.
I'm not seeing anything from any of these linux clients (debian and redhat based) with respect to the the beats applications installed. 

Soquery has been installed on nearly all of the linux clients successfully as well, but those are reporting in to Fleet without any issues. So, there doesnt seem to be a networking issue.

**Comment by cm-ops** [comment]
If you have verified the firewall settings and network connectivity, you can check the manager Logstash pipelne to look at the beats input and see if there are drops there - `so-logstash-pipeline-stats manager`

**Reply by ll3N1GmAll** [reply]
Is it safe to post the output of running that command here? I don't see anything that jumps out as sensitive; but want to ask before I potentially do something foolish. I don't see anything in that output that indicates failures.

**Reply by cm-ops** [reply]
Just looking to see if there are drops from the `beats input`. Are there?
